src/System.ComponentModel.TypeConverter/src/System/ComponentModel/AttributeCollection.cs

```diff
@@ -378,6 +378,19 @@ object ICollection.SyncRoot
             }
         }
 
+        int ICollection.Count 
```

Comment:
One of the [requirements](https://github.com/dotnet/standard/tree/master/docs/netstandard-20#requirements) for netstandard2.0 is that we are going to bring back types wholesale and not break them up. If we believe the type is meaningful for netstandard2.0 we have to port it over with all its members.

In this case, the types are in netstandard2.0 but they were missing some APIs (i.e. explicit interface implementations).